TIG/TIG/Feature/Home/TimerAnnounceView.swift

```diff
@@ -8,22 +8,24 @@
 import SwiftUI
 
 struct TimerAnnounceView: View {
+    @Environment(HomeViewModel.self) var homeViewModel
+
     var body: some View {
         VStack(alignment: .center) {
             VStack {
                 Image("AvailableIcon")
```

Comment:
Image 컴포넌트에 리터럴 값을 넣어주는 것보다 resource 파라미터를 사용해 주시는 것이 좋아 보입니다.
리터럴 값으로 넣어주면 image asset 이름이 변경되었을 때 컴파일 오류 없이 실행이 되지만, 앱 화면에서는 아무 이미지가 보이지 않게 됩니다.
resource 파라미터를 사용하게 되면 image asset 이름이 변경되었을 때, 컴파일 오류가 발생해 문제 지점을 확인할 수 있습니다.
```suggestion
                Image(.availableIcon)
```

Reply:
했음요